server_development/topics/providers.adoc

```diff
@@ -215,7 +183,6 @@ The provider ID must match the ID returned by the `ProviderFactory.getId()` of t
 particular provider factory implementation. Some provider types can be retrieved with the usage of `ComponentModel` as the second argument and some (for example `Authenticator`) even
 need to be retrieved with the usage of `KeycloakSessionFactory`. It is not recommended to implement your own providers this way as it may be deprecated in the future.
 
-ifeval::["{kc_dist}" == "quarkus"]
 === Registering provider implementations
```

Comment:
We can remove or reference https://www.keycloak.org/server/configuration-provider.

Reply:
Focus here is to remove the WildFly documentation, not to enhance the Quarkus documentation. So this should be handled in a follow-up.